src/routing/routing.component.test.tsx

```diff
@@ -115,6 +116,32 @@ describe('Routing component', () => {
         admin: true,
         order: 5,
       },
+      {
+        section: 'test section',
+        link: 'test link not authorised',
+        plugin: 'test_plugin_name',
+        displayName: 'Test Plugin Not Authorised',
+        unauthorised: true,
+        order: 6,
+      },
+    ];
+
+    const { asFragment } = render(<Routing />, { wrapper: Wrapper });
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('renders an unauthorised route for a plugin', () => {
+    state.scigateway.authorisation.provider = new TestAuthProvider('logged in');
```

Comment:
Surely to double check that unauthorised routes work right we shouldn't "log in" for this test?

Reply:
Yes, my mistake 😅